CRM/Member/Form/Membership.php

```diff
@@ -1793,9 +1772,9 @@ protected function legacyProcessRecurringContribution(array $params, $contactID)
     }
     $recurParams = ['contact_id' => $contactID];
     $recurParams['amount'] = $this->order->getTotalAmount();
-    $recurParams['auto_renew'] = $params['auto_renew'] ?? NULL;
-    $recurParams['frequency_unit'] = $params['frequency_unit'] ?? NULL;
-    $recurParams['frequency_interval'] = $params['frequency_interval'] ?? NULL;
+    $recurParams['auto_renew'] = TRUE;
```

Comment:
@eileenmcnaughton I thought your comment in the PR was helpful:

```suggestion
    // for the legacyProcessRecurringContribution function to be reached auto_renew must be true
    $recurParams['auto_renew'] = TRUE;
```